### PR TITLE
YLP-684 URL Mapping

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

File: packages/yourloops/components/patient-data.tsx [hunk 1]
```diff
@@ -71,6 +75,7 @@ function PatientDataPage(): JSX.Element | null {
   const { blipApi } = dataHook;
   const { patientId: paramPatientId = null } = paramHook as PatientDataParam;
   const userId = authHook.user?.userid ?? null;
+  const prefixURL = authHook.user?.role === UserRoles.patient ? props.prefixURL : `${props.prefixURL}/${paramPatientId}`;
```
Comment (franck-fourel) [inline packages/yourloops/components/patient-data.tsx:78]: why don't we put as well the userid for patient

**Review by franck-fourel — CHANGES_REQUESTED** [review]
works fine but still a couple of comments

I noticed that the link to the profile modal does not work when clicking it from the Settings page 
image

The `/hcp/ path is still used in `getPatientLink()` in /blip/app/components/chart/header.js

**Comment by yvon-dblg** [comment]
> I noticed that the link to the profile modal does not work when clicking it from the Settings page

For some reason, it is apparently the wanted behavior. But I didn't find where it comes from.
I just recall asking Leo about that too.

> The `/hcp/ path is still used in `getPatientLink()` in /blip/app/components/chart/header.js
👍 I will fix it, but it will fully fixed in https://github.com/mdblp/blip/pull/489
Given the changes made in this PR it will be easier.

> why don't we put as well the userid for patient
My bad. I didn't read well the document.

**Review by franck-fourel — APPROVED** [review]
LGTM

**Pull request merged**